platform/feature-api/api/src/pod-api/api.ts

```diff
@@ -39,6 +40,7 @@ export interface Matcher {
  * on, except for internal purposes of the Feature.
  */
 export interface PolyIn {
+    store: unknown;
```

Comment:
Any reason this is in the interface? Are features meant to to use the store directly? Otherwise, I'd remove the `store` property from the interface and make it private in the class.

Reply:
No was just hacking it together to make it work there, feel free to remove it from the interface